### PR TITLE
API-1372: Doc Events API Format

### PR DESCRIPTION
**Pull request by tomglvng**

### CODE REVIEW

**Comment by meganelepalud-akeneo** [comment]
hello, can you rebase your branch with events-api-release? 
I made the overview today.  
https://github.com/akeneo/pim-api-docs/pull/440 

request header and request body could be added to content/events-api/receiving-events.md

File: content/events-api/overview.md [hunk 1]
```diff
@@ -4,4 +4,45 @@ _All the essential things you need to know._
...
+| Name | Example | Format |
+| ---- | ------- | ------ |
+| x-akeneo-request-signature | `d4582dd13a7b654c212d70b4c705e0ab0c0e6eb7f6524080f4d39121f88e6061` | |
+| x-akeneo-request-timestamp | `1602520454` | Unix timestamp in seconds |
+
+### Request body
+
+| Name | Description | Example | Format |
+| ---- | ----------- | ------- | ------ |
+| action | The action attribute helps you identify the event type you received, which action has been done in the PIM. | `product.created` | `[resource_name].[event_name]` |
```
Comment (fitn) [inline content/events-api/overview.md:42]:
```suggestion
| action | The action property helps you identify the API event type you received, which action has been done in the PIM. | `product.created` | `[resource_name].[event_name]` |
```

File: content/events-api/overview.md [hunk 2]
```diff
@@ -4,4 +4,45 @@ _All the essential things you need to know._
...
+| event_id | The event_id is the unique identifier of the event. It allows you to be sure that you received the same event only one time. | `6ad821d8-2468-4632-89d2-92e442c72313` | UUID V4|
```
Comment (fitn) [inline content/events-api/overview.md:43]:
```suggestion
| event_id | The event_id is the unique identifier of the event. It allows you to ensure you received the same API event only one time. | `6ad821d8-2468-4632-89d2-92e442c72313` | UUID V4|
```

File: content/events-api/overview.md [hunk 3]
```diff
@@ -4,4 +4,45 @@ _All the essential things you need to know._
...
+| event_date | The event_date contains the date and time of the event. | 2020-01-01T00:00:00+00:00 | Format ISO 8601 |
```
Comment (fitn) [inline content/events-api/overview.md:44]: So that may be better to rename it event_datetime?

File: content/events-api/overview.md [hunk 4]
```diff
@@ -4,4 +4,45 @@ _All the essential things you need to know._
...
+| author | The author attribute tells you who performed the action. | `julia` or `magento_0000` | |
+| author_type | The author_type is the type of user who performed the action. It can be a UI user, like Julia, or a REST API call coming from another application. | `ui` or `api` | |
```
Comment (fitn) [inline content/events-api/overview.md:46]: What about backend tasks like rule engine?

File: content/events-api/overview.md [hunk 5]
```diff
@@ -4,4 +4,45 @@ _All the essential things you need to know._
...
+| pim_source | The pim_source attribute helps you to identify which Akeneo PIM sent the event. | `staging.akeneo.com` | From `AKENEO_PIM_URL` environment variable |
```
Comment (fitn) [inline content/events-api/overview.md:47]:
```suggestion
| pim_source | The pim_source property helps you to identify which Akeneo PIM sent the event. | `staging.akeneo.com` | From `AKENEO_PIM_URL` environment variable |
```

File: content/events-api/overview.md [hunk 6]
```diff
@@ -4,4 +4,45 @@ _All the essential things you need to know._
...
+| x-akeneo-request-timestamp | `1602520454` | Unix timestamp in seconds |
+
+### Request body
+
+| Name | Description | Example | Format |
+| ---- | ----------- | ------- | ------ |
+| action | The action attribute helps you identify the event type you received, which action has been done in the PIM. | `product.created` | `[resource_name].[event_name]` |
+| event_id | The event_id is the unique identifier of the event. It allows you to be sure that you received the same event only one time. | `6ad821d8-2468-4632-89d2-92e442c72313` | UUID V4|
+| event_date | The event_date contains the date and time of the event. | 2020-01-01T00:00:00+00:00 | Format ISO 8601 |
+| author | The author attribute tells you who performed the action. | `julia` or `magento_0000` | |
```
Comment (fitn) [inline content/events-api/overview.md:45]:
```suggestion
| author | The author property tells you who performed the action. | `julia` or `magento_0000` | |
```

File: content/events-api/overview.md (hunk 4 above)
Comment (tomglvng) [reply content/events-api/overview.md:46]: All non-API users will have author_type = 'UI' so, backend tasks will have 'UI' as author_type.

File: content/events-api/overview.md (hunk 3 above)
Comment (tomglvng) [reply content/events-api/overview.md]: Yes, done in https://github.com/akeneo/pim-api-docs/pull/439/commits/dddc3eee107bbe8d7585fbf88ae094b90565c6b1

**Review by pierrejolly — APPROVED** [review]

**Review by willy-ahva — APPROVED** [review]

**Pull request merged**